easybuild/easyconfigs/r/Racon/Racon-1.4.7-fosscuda-2019b.eb

```diff
@@ -0,0 +1,163 @@
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/logger/archive'],
+        'filename': 'logger.zip',
+        'download_filename': 'b88c8ee8ba59fd07b40c022957b7bdce1a5f8502.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/rampler/archive'],
+        'filename': 'rampler.zip',
+        'download_filename': 'a6dec0140878ba81da90387c77740756fefa4920.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/spoa/archive'],
+        'filename': 'spoa.zip',
+        'download_filename': '11219575bdcb356c57fb32f54cf23b8da2abc4f0.zip',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/thread_pool/archive'],
+        'filename': 'thread_pool.zip',
+        'download_filename': '048da9485390ee8b8a7ecedccc598a06ca4429e2.zip',
+    },
+    {
+        'source_urls': ['https://github.com/madler/zlib/archive'],
+        'filename': 'zlib.zip',
+        'download_filename': 'cacf7f1d4e3d44d871b605da3b647f07d718623f.zip',
+    },
+    {
+        'source_urls': ['https://github.com/google/benchmark/archive'],
+        'filename': 'benchmark.zip',
+        'download_filename': '090faecb454fbd6e6e17a75ef8146acb037118d4.zip',
+    },
+    {
+        'source_urls': ['https://github.com/gabime/spdlog/archive'],
+        'filename': 'spdlog.zip',
+        'download_filename': '366935142780e3754b358542f0fd5ed83793b263.zip',
+    },
+
+]
+checksums = [
+    '7d59c310eb8e7472f6b7bb523206cb0642fe7db32335011cd6b9e7b31cd549ee',  # 1.4.7.tar.gz
+    '228391b8b43effb286cad814ae281c57fad5fcca1724c926cf4cd9ad46b88352',  # ClaraGenomicsAnalysis.zip
+    '1bd393939fd7db6930270cc159545081a2350c89e990affb3ed016f7ae29703c',  # bioparser.zip
+    '6ab3779306b16ece3b42cfcbb3b9236b160651bb8130ac9903c9c5601eee1323',  # edlib.zip
+    '3be3694b5483970accd9b9ea79cd6f05ba347342afea42ef8e180ae112ff9600',  # googletest.zip
+    '9a695a12410b24da6c27f18fcdd95946405fecb26e66cee076f503dd5a34ad82',  # logger.zip
+    '21b99b519438593be6fe3f8e6300c5f9c299e7ee3e5de136299edf3b1591b2ae',  # rampler.zip
+    '3f086337f40b201627cf2fce170365496575b85dda5606a2d78f34aa5e8cc3dc',  # spoa.zip
+    '0af714c760b20c648ee049e6ff6ed4b70d1d0f093469550f42568bc9123c86a8',  # thread_pool.zip
+    '1cce3828ec2ba80ff8a4cac0ab5aa03756026517154c4b450e617ede751d41bd',  # zlib.zip
+    'e777f978593ea6db38356ce09ec3902e839b3037a9a19ff543e6f901e50cc773',  # benchmark.zip
+    'f8833ab5af932a7b0315a5c3d527a748d47839d1a8ac72ab7af9fd7277498554',  # spdlog.zip
+]
+
+builddependencies = [('CMake', '3.15.3')]
+
+# Populate first level of git submodules
+# 1. remove submodule dir
+# 2. move downloaded source to removed submodule dir
+#
+# ClaraGenomicsAnalysis
+preconfigopts = "rmdir %(builddir)s/racon-gpu-%(version)s/vendor/ClaraGenomicsAnalysis && " \
+                "mv %(builddir)s/ClaraGenomicsAnalysis-416af9f1817a4a70745b3f7cdb7418125159f75c " \
+                "%(builddir)s/racon-gpu-%(version)s/vendor/ClaraGenomicsAnalysis && "
```

Comment:
You can avoid using `rmdir` (just move the contents), and you can make this more generic using `*` to avoid having to change the commit ID in two places if we update this later:

```
local_vendor_dir = '%(builddir)s/racon-gpu-%(version)s/vendor/'
preconfigopts = "mv %%(builddir)s/ClaraGenomicsAnalysis-*/* %s/ClaraGenomicsAnalysis/ && " % local_vendor_dir
```

Reply:
Does this moves dot files? Not sure if needed, just wondering.

Reply:
It doesn't, but I doubt those are needed? Is this just about the `.git` dir?

Reply:
I doubt too, but I shot myself to foot few times copying stuff like that. Fortunately this is not the case :)